squad-dedicated-serverconfig.json

```diff
@@ -228,7 +242,7 @@
     {
         "DisplayName": "Record Demos",
```

Comment:
Just a recommendation, should this be renamed to reflect server side like the description?

Reply:
Have reworded both demo settings for better readability and descriptiveness